src/main/java/org/fogbowcloud/manager/core/plugins/identity/ldap/LdapIdentityPlugin.java

```diff
@@ -84,7 +84,6 @@ public LdapIdentityPlugin(Properties properties) {
 		this.publicKeyPath = properties.getProperty(PUBLIC_KEY_PATH);
 	}
 
-	@Override
```

Comment:
Any reason for removing the annotation @Override?

Reply:
Wow, I never touched `LdapIdentityPlugin` class. Maybe Intellij made that, but anyway I'll fix it.